src/test/debugger/appWorker.test.ts

```diff
@@ -99,6 +99,19 @@ suite("appWorker", function() {
                     assert(postReplyFunction.calledWith({ data: testMessage }), "No echo back from app");
                 });
             });
+
+            test("should be able to require an installed node module via __debug__.resolve", function () {
+                const expectedMessageResult = { success: true };
+                const startScriptContents = `__debug__.require('q');

```

Comment:
Can you get something from the q module, and send it on the testResponse so we can validate that the proper q module was load? (Currently the test would pass if __debug__.require is a no-op)